core/templates/components/state-editor/state-interaction-editor/state-interaction-editor.component.ts

```diff
@@ -254,6 +255,10 @@ export class StateInteractionEditorComponent
     });
   }
 
+  toggleBtn(): void {
+    this.showToggleBtn = !this.showToggleBtn;
+  }
```

Comment:
Add a test when `showToggleBtn` then changed to false also.

Reply:
it is false originally my mistake

Reply:
Ah, sorry I wasn't clear in my first comment. I meant to add a frontend test to check if the `showToggleBtn` is `true` and then changes to `false` when the appropriate function is called.